app/Gists/GistConfig.php

```diff
@@ -0,0 +1,83 @@
+<?php namespace Gistlog\Gists;
+
+use Illuminate\Support\Arr;
+use Symfony\Component\Yaml\Yaml;
+
+class GistConfig implements \ArrayAccess
+{
+
+    /**
+     * @var array
+     */
+    private $default_settings = array(
```

Comment:
I think we've done camelCase everywhere else in the app, both for properties and for inline variables